programs/stake/src/stake_instruction.rs

```diff
@@ -5866,4 +5866,24 @@ mod tests {
         // Both fully deactivated works
         try_merge(transaction_accounts, instruction_accounts, Ok(()));
     }
+
+    #[test]
+    fn test_stake_get_minimum_delegation() {
+        let stake_address = Pubkey::new_unique();
+        let stake_account = create_default_stake_account();
+        let instruction_data = serialize(&StakeInstruction::GetMinimumDelegation).unwrap();
+        let transaction_accounts = vec![(stake_address, stake_account)];
+        let instruction_accounts = vec![AccountMeta {
+            pubkey: stake_address,
+            is_signer: false,
+            is_writable: false,
+        }];
+
+        process_instruction(
```

Comment:
How about checking the return value as well?

E.g. (untested):
```Rust
mock_process_instruction(
    &id(),
    Vec::new(),
    instruction_data,
    transaction_accounts,
    instruction_accounts,
    None,
    Ok(()),
    |first_instruction_account: usize, invoke_context: &mut InvokeContext| {
        let result = super::process_instruction(first_instruction_account, invoke_context);
        assert_eq!(
            invoke_context.transaction_context.get_return_data().1,
            Vec::from(minimum_delegation.to_le_bytes()),
        );
        result
    },
)
```

Reply:
Oh, this is great! Will do.

Reply:
Done!